packages/sources/therundown/test/integration/schedule.ts

```
@@ -38,7 +65,32 @@ export function scheduleTests(): void {
     })
 
     describe('when successfully requesting Therundown API', () => {
```

Both "malformed" tests should be inside the same `describe` (describe 2):
- describe1: `error calls`
  - describe 2: `when successfully requesting Therundown API`:
    - test 1: `should throw an exception if a game attribute is malformed while creating market`
    - test 2: `should throw an exception if a game attribute is malformed while resolving market`

There are many ways to organize them but given describe 2 it make sense to have them both within.

---

Resolved